app/forms/idv/consent_form.rb

```diff
@@ -3,7 +3,7 @@ class ConsentForm
     include ActiveModel::Model
 
     validates :ial2_consent_given?,
-              acceptance: { message: Proc.new { I18n.t('errors.doc_auth.consent_form') } }
+              acceptance: { message: proc { I18n.t('errors.doc_auth.consent_form') } }
```

Comment:
We have a custom cop that encourages `Proc.new` when it checks for i18n'd messages, can we update its suggestion? 

https://github.com/18F/identity-idp/blob/f35242b670f887a3cb8721c9d026fde2cc33468d/lib/linters/localized_validation_message_linter.rb#L5

Reply:
fixed in 3307f5a96